README.md

```diff
@@ -68,17 +68,16 @@ See the [install instructions](/docs/introduction/install.md)
 
 ## Contributing
 
-Please abide by the [Code of Conduct](CODE_OF_CONDUCT.md) in all interactions,
```

Comment:
we need to preserve section for non key contributors (i.e. if it's my first/second PR, what things I should read/know).

Reply:
I'm not so clear on what you mean, this entire section is basically for people looking at getting started with contributing to the project

Reply:
I think Anton just means we need to keep a link to the code of conduct in here in addition to everything else (and I agree).

Reply:
Okay, yes, I just moved it to the bottom of the contributors section, the thinking being that most people would already know what good conduct is, so perhaps best to start with the guide which might contain important information that they are not aware of yet

Reply:
> the thinking being that most people would already know what good conduct is 

Ideally, yes, but the members of various open source community have proven themselves again and again to have absolutely no idea what appropriate conduct is